### PR TITLE
LPS-105060 New test to check isolation across companies in OAuth2 without any scopes

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by csierra** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/oauth2-provider/oauth2-provider-shortcut/src/main/java/com/liferay/oauth2/provider/shortcut/internal/instance/lifecycle/FragmentRendererPortalInstanceLifecycleListener.java [hunk 1]
```diff
@@ -64,7 +63,7 @@ public void portalInstanceRegistered(Company company) throws Exception {
 		User user = _userLocalService.getDefaultUser(company.getCompanyId());
 
 		oAuth2Application = _oAuth2ApplicationLocalService.addOAuth2Application(
-			company.getCompanyId(), user.getUserId(), user.getScreenName(),
```
Comment (csierra) [inline modules/apps/oauth2-provider/oauth2-provider-shortcut/src/main/java/com/liferay/oauth2/provider/shortcut/internal/instance/lifecycle/FragmentRendererPortalInstanceLifecycleListener.java]: hey @martamedio, 
sorry this review took so long...
we should be setting the ThreadLocal in this places now, shouldn't we? I would be curious to know actually what company has the CompanyThreadLocal set in this case, if any...

Comment (csierra) [reply modules/apps/oauth2-provider/oauth2-provider-shortcut/src/main/java/com/liferay/oauth2/provider/shortcut/internal/instance/lifecycle/FragmentRendererPortalInstanceLifecycleListener.java]: nevermind about this... it turns out it is set here: https://github.com/liferay/liferay-portal/blob/5c25dfdb42064c234dea22c3987b3253d4e1da36/modules/apps/portal/portal-instance-lifecycle-impl/src/main/java/com/liferay/portal/instance/lifecycle/internal/PortalInstanceLifecycleListenerManagerImpl.java#L138

but we don't set it when the company is removed, so we would need to send a patch to that... even though we are nout using it yet, but to make it consistent

**Comment by csierra** [comment]
Pull request submitted to https://github.com/martamedio/liferay-portal/pull/118. [See changes here.](https://github.com/csierra/liferay-portal/compare/2f4b0c598b02~1...2f4b0c598b02) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**